docs/Examples/miner-configs.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const bcoin = require('../..');
+bcoin.set('regtest');
```

Comment:
While `bcoin.set()` is used in other examples, it's depending on global state, which often desired to be avoided, so it could be good to include passing regtest to chain here.